aptos-move/framework/aptos-framework/sources/genesis.move

```diff
@@ -126,6 +146,53 @@ module aptos_framework::genesis {
         aptos_coin::configure_accounts_for_test(aptos_framework, &core_resources, mint_cap);
     }
 
+    fun create_accounts(aptos_framework: &signer, accounts: vector<AccountMap>) {
+        let i = 0;
+        let num_accounts = vector::length(&accounts);
+        let unique_accounts = vector::empty();
+
+        while (i < num_accounts) {
+            let account_map = vector::borrow(&accounts, i);
+            assert!(
+                !vector::contains(&unique_accounts, &account_map.account_address),
+                error::already_exists(EDUPLICATE_ACCOUNT),
+            );
```

Comment:
Probably would have been easier to just check if the account existed rather than keep track of already created accounts

Reply:
hahahaha maybe, I can update that